### PR TITLE
[FIX] point_of_sale: Customer Display Corrections

### PR DESCRIPTION
**Pull request by vlst-odoo**
[FIX] point_of_sale: Customer Display Corrections

Currently, the customer display has a few misleading behaviors.

1. Display doesn't reset when expected

After successful payment, the details of the previous order remain visible until the 1st product of the next order is encoded —> We now show a _Thank You_ Message after successful payment. 

2. Discount lines are not shown in the Customer Display

Currently, when a discount is applied to the whole order, a new line appears, showing this discount, but if a discount is applied to a specific product, there is no indication on the Customer Display —> we now also display a discount line under the product name for each product where a specific discount is applied.

Link to task:
https://www.odoo.com/web#id=2906039&cids=1&menu_id=4720&action=333&active_id=1737&model=project.task&view_type=form

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by trj-odoo** [comment]
This'll be the first thing I'll give you: https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html?highlight=github%20guideline#commit-message-structure
Always try to follow this when commiting

File: doc/cla/individual/vlst-odoo.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Belgium, 1 Dec 2022
```
Comment (trj-odoo) [inline doc/cla/individual/vlst-odoo.md:1]: No need, you're part of odoo

File: addons/point_of_sale/static/src/xml/CustomerFacingDisplay/CustomerFacingDisplayOrder.xml [hunk 2]
```diff
@@ -25,48 +25,85 @@
...
+                            <div t-esc="orderline.get_quantity_str()"/>
+                            <div t-esc="pos.format_currency(orderline.get_display_price())"/>
+                        </div>
                     </div>
                 </t>
-            </div>
+                <!-- after the order has been validated -->
+                <!-- we show the 'thank you' message -->
+                <t t-else=""> 
+                    <div class="pos-thank_you_message">
```
Comment (trj-odoo) [inline addons/point_of_sale/static/src/xml/CustomerFacingDisplay/CustomerFacingDisplayOrder.xml:102]: don't mix kebab and snake case

Comment (vlst-odoo) [reply addons/point_of_sale/static/src/xml/CustomerFacingDisplay/CustomerFacingDisplayOrder.xml]: Changed class name to "pos-thank-you-message"

**Pull request closed without merging**